### PR TITLE
New package: libclang-python3-3.9.0

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: srcpkgs/libclang-python3/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+version=3.9.0
+revision=1
+wrksrc="libclang-py3-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3 python3-setuptools"
+short_desc="Python3 bindings for libclang"
+maintainer="shizonic <realtiaz@gmail.com>"
+homepage="https://github.com/Sublimelibclang-py3/libclang-py3"
```
Comment (maxice8) [inline srcpkgs/libclang-python3/template]: This page doesn't exist


This is the page referenced on PYPI as homepage

https://bitbucket.org/Anteru/python3-libclang

File: srcpkgs/libclang-python3/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+license="Unknown"
```
Comment (maxice8) [inline srcpkgs/libclang-python3/template]: setup.py says BSD license (which doesn't exist, there lots of BSD-style licenses) but the files have `NCSA` header

Comment (shizonic) [reply srcpkgs/libclang-python3/template]: So that means? Which license should I use?

File: srcpkgs/libclang-python3/template (hunk 1 above)
Comment (shizonic) [reply srcpkgs/libclang-python3/template]: Will change that asap.

File: srcpkgs/libclang-python3/template (hunk 2 above)
Comment (shizonic) [reply srcpkgs/libclang-python3/template]: I can't find any license file in the repo. So do I have to create it by myself?

File: srcpkgs/libclang-python3/template (hunk 1 above)
Comment (shizonic) [reply srcpkgs/libclang-python3/template]: Fixed.

File: srcpkgs/libclang-python3/template (hunk 2 above)
Comment (Cogitri) [reply srcpkgs/libclang-python3/template]: Hm, the sources files say `This file is distributed under the University of Illinois Open Source License.`, sooooo

**Comment by Cogitri** [comment]
Ah, yes, my bad.

**Comment by shizonic** [comment]
@Cogitri NP ;)

File: srcpkgs/libclang-python3/template (hunk 2 above)
Comment (shizonic) [reply srcpkgs/libclang-python3/template]: Okay. Will do that. Thanks!

**Comment by maxice8** [comment]
This package claims compatibility against clang 3.9

we are on clang 6.0.0 and we walk in lockstep with llvm, so after crystallang get their stuff together, @cogitri will be moving to llvm 7.

I see no reason to add this package unless it decides to walk in lockstep with llvm/clang

**Comment by maxice8** [comment]
Sorry, last commit 2 years ago, and it aims for compatibility llvm3.9 which we are removing.

**Pull request closed without merging**